packages/vats/src/core/chain-behaviors.js

```diff
@@ -382,6 +383,79 @@ export const makeChainStorage = async ({
   chainStorageP.resolve(rootNodeP);
 };
 
+/**
+ * @typedef HighPrioritySendersManager
+ * @property {(namespace: string, address: string) => Promise<void>} add
+ * @property {(namespace: string, address: string) => void} remove
+ */
+export const PRIORITY_SENDERS_NAMESPACE_RE = /^[a-zA-Z0-9_-]{1,50}$/;
+
+/**
+ * @param {BootstrapSpace & {
+ *   consume: { loadCriticalVat: ERef<VatLoader<ChainStorageVat>> }
+ * }} powers
+ */
+export const makeHighPrioritySendersManager = async ({
+  consume: { loadCriticalVat, bridgeManager: bridgeManagerP },
+  produce: { highPrioritySendersManager: managerP },
+}) => {
+  const bridgeManager = await bridgeManagerP;
+  if (!bridgeManager) {
+    console.warn('Cannot support chainStorage without an actual chain.');
+    managerP.resolve(null);
+    return;
+  }
+
+  const storageBridgeManager = E(bridgeManager).register(BRIDGE_ID.STORAGE);
```

Comment:
Wouldn't this conflict with the same registration in `makeChainStorage` ? I think it needs to be produced by `makeChainStorage` so it can be shared with this.